### PR TITLE
make sure to create a full-fledged adb instance

### PR DESCRIPTION
**Pull request by jlipps**
and pass it the new androidAdbPort cap

### CODE REVIEW

File: lib/driver.js [hunk 1]
```diff
@@ -98,8 +97,16 @@ class SelendroidDriver extends BaseDriver {
...
+
+    // get device udid for this session
+    let {udid, emPort} = await helpers.getDeviceInfoFromCaps(this.opts);
+    this.opts.udid = udid;
+    this.opts.emPort = emPort;
+
+    // now that we know our java version and device info, we can create our
+    // ADB instance
+    this.adb = await androidHelpers.createADB(this.opts.javaVersion,
+        this.opts.udid, this.opts.emPort, this.opts.androidAdbPort);
```
Comment (imurchie) [inline lib/driver.js]: This ought to be `adbPort`, as in Android.

**Comment by imurchie** [comment]
Does the `appium-adb` version need to be updated in `package.json`? Otherwise, looks good!

**Comment by jlipps** [comment]
it does. as does probably the appium-android-driver version. will update them both after they have both been published.

**Comment by jlipps** [comment]
published as 1.2.1

**Pull request merged**